### PR TITLE
Logical and

### LINKED ISSUE #17734: logical_and
implementing the logical and to the paddle frontend

### LINKED ISSUE #18214: logical_and
adding logical and to paddle frontend

### PR DESCRIPTION
**Pull request by samthakur587**
Close #17734
Close #18214

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py [hunk 1]
```diff
@@ -468,3 +468,31 @@ def test_paddle_isclose(
...
     )
+
+
+# logical_and
+@handle_frontend_test(
+    fn_tree="paddle.logical_and",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), num_arrays=2, shared_dtype=True
+    ),
+    test_with_out=st.just(True),
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py:479]: No need for this. It's true by default.

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @thecoder12! Thanks for opening this PR. Would you please resolve the merge conflict? Feel feel to request another review when you're done. Thanks :)

**Comment by samthakur587** [comment]
hey @hello-fri-end  i made all changes that are required. and all test are passing in my local. now you can check is there any changes required. 

Now its ready to merge.

thank you

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py (hunk 1 above)
Comment (samthakur587) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py]: yeah ,I removed this from here. now its working perfect.

**Comment by hello-fri-end** [comment]
> hey @hello-fri-end i made all changes that are required. and all test are passing in my local. now you can check is there any changes required.
> 
> Now its ready to merge.
> 
> thank you

Hey @samthakur587 ! It looks like you are trying to add both the frontend function and the instance method for `logical_and` in this PR. Currently, there's an open task for adding `logical_and` as a frontend function unassigned in this ToDo list (https://github.com/unifyai/ivy/issues/15036). Can you please assign that to yourself so we can link it to this PR? Also,  In the future, it's better to open separate PRs for different tasks. Feel free to request another review when you are done. Thanks :)

**Comment by samthakur587** [comment]
hey @hello-fri-end now i assign the subtask to me and solved the merge. i implemented just to call the method from  paddle  frontend instande of calling from ivy frontend. i had never seen that it is a different sub task.

thank you

**Comment by hello-fri-end** [comment]
> hey @hello-fri-end now i assign the subtask to me and solved the merge. i implemented just to call the method from paddle frontend instande of calling from ivy frontend. i had never seen that it is a different sub task.
> 
> thank you

LGTM! Thanks for the contribution :rocket:

**Pull request merged**